document_page_approval/views/document_page_approval.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<openerp>
+<odoo>
     <data>
```

Comment:
It seems no more data tag required in v9, as said A Lesuisse.
https://github.com/odoo/odoo/blob/9.0/addons/calendar/views/calendar.xml
then readibility of xml is better (4 chars won).